include/attribute.h

```diff
@@ -270,6 +256,7 @@ struct SaveBlock1
     u8 filler5C8[0xE];
     u16 fishingPoint;
 }; /* size = 0x5D8 */
+/* TODO: list offsets */
```

Comment:
General advice: it's weird to add a comment below. 
Usually we do it above what you want to comment on.